backpack.sh

```diff
@@ -81,34 +85,39 @@ unit-test-browser-refresh () {
 	clean
 	trap stop EXIT
 	open http://localhost:8091/
-	npx shadow-cljs watch browser
+	shadow-cljs watch browser
 	abort_on_error
 }
 
 unit-test-cljs-refresh () {
 	clean
 	echo_message 'In a few special places, these clojure changes create some of the greatest transformation spectacles on earth'
-	npx shadow-cljs compile karma
+	shadow-cljs compile karma
 	abort_on_error
 	trap stop EXIT
 	npx karma start --no-single-run &
-	npx shadow-cljs watch karma
+	shadow-cljs watch karma
 }
 
 ## unit-test-cljs:
-## args: [-b|-n|-r]
+## args: [-k|-b|-n|-r] [test-ns-regex]
 ## Runs the ClojureScript unit tests
+## [-k] Watches and compiles tests for execution with karma (Default)
 ## [-b] Watches and compiles tests for execution within a browser
 ## [-n] Executes the tests targeting Node.js
-## [-r] Watches tests and source files for changes, and subsequently re-evaluates
+## [-r] Watches tests and source files for changes, and subsequently re-evaluates with karma
+## [test-ns-regex] Watches tests and source files for changes, and subsequently re-evaluates
 unit-test-cljs () {
+	export TEST_NS_REGEXP=${2:-'-test$'}
```

Comment:
I tried an approach where we had a binding and closure define in the test runner to filter which NS to evaluate... That only works for the browser tests, and not the nodejs/karma ones.

This approach works for all test modes